docs/cybersecurity/SSH-attack/manual.sql

```diff
@@ -1,9 +1,9 @@
 INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-07-14T00:10:10.300Z', 'asgard.example.com', 0, 'Line protocol on Interface GigabitEthernet0/1, changed state to up', NULL);
 INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-05T20:11:13.138Z', 'asgard.example.com', 0, 'partition health measures for /var did not suffice - still using 96% of partition space', NULL);
-INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:11.333Z', 'asgard.example.com', 4, 'Invalid user asmith', '192.168.10.83');
-INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:12.333Z', 'asgard.example.com', 4, 'Invalid user bsmith', '192.168.10.83');
+INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:11.333Z', 'asgard.example.com', 4, 'Invalid user asmith', '192.168.10.81');
+INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:12.333Z', 'asgard.example.com', 4, 'Invalid user bsmith', '192.168.10.82');
 INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:13.333Z', 'asgard.example.com', 4, 'Invalid user csmith', '192.168.10.83');
-INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:14.333Z', 'asgard.example.com', 4, 'Invalid user dsmith', '192.168.10.83');
-INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:15.333Z', 'asgard.example.com', 4, 'Invalid user esmith', '192.168.10.83');
-INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:16.333Z', 'asgard.example.com', 4, 'Invalid user fsmith', '192.168.10.83');
+INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:14.333Z', 'asgard.example.com', 4, 'Invalid user dsmith', '192.168.10.84');
+INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:15.333Z', 'asgard.example.com', 4, 'Invalid user esmith', '192.168.10.85');
+INSERT INTO syslog (ts, host, facility, message, remote_address) VALUES ('2021-08-08T02:11:16.333Z', 'asgard.example.com', 4, 'Invalid user fsmith', '192.168.10.86');
```

Comment:
Why those changes?

Reply:
For variability and more realistic src IP address spoofing, but not necessary.  In fact, maybe not good if it _should_ be consistent with host being the same.  I can revert

Reply:
I don't now :) Was just wondering.

If @rspurgeon did the original recipe it should be his call.

Reply:
My thought was an attacker was trying different user names from the same source IP... but I'm indifferent here.

Reply:
I'll revert